chopper/lib/src/base.dart

```diff
@@ -248,10 +244,10 @@ class ChopperClient {
   ) async {
     var error = response.body;
     if (errorConverter != null) {
-      final errorRes = await errorConverter.convertError<BodyType, InnerType>(
+      final errorRes = await errorConverter?.convertError<BodyType, InnerType>(
```

Comment:
do we really need to do `errorConverter?` here since we check `errorConverter != null` before ? (just a curiosity question)

Reply:
Unfortunately, yes. Although we got null safety in Dart, we didn't get auto-cast with it... Another solution could be `errorConverter!`.

Reply:
Cool. Keet is without changes for now.